### DISCUSSION TITLE
IDE for Observable?

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#1512] [Q&A] [answered]
What is the recommended editor for editing qmd files containing ojs code blocks?

**Comment by mcanouil** [comment]
I would have assume VSCode with Quarto extension but it appears beyond code highlighting, you can't run cells.
This is definitely something I would like to be able to do in VSCode.
<img width="329" alt="image" src="https://user-images.githubusercontent.com/8896044/180655002-3bb285d2-feb5-425b-8875-08cec5efd723.png">

**Comment by jjallaire** [comment, marked as answer]
Yes, OJS tooling right now requires a full render (in both RStudio and VS Code). Definitely something we will improve on!

**Reply by eitsupi** [reply]
Thanks for the reply. Perhaps this information should be in the documentation?
